### PR TITLE
fix(license): minor code fix

### PR DESCRIPTION
**Pull request by rajaSahil**
- Minor code fix for license

### CODE REVIEW

**Review by nyrahul — APPROVED** [review]
LGTM

File: src/license/license.go [hunk 1]
```diff
@@ -75,6 +76,15 @@ func CheckLicenseSecret() error {
 	return nil
 }
 
+func (t *Token) checkTokenExpiration() error {
+	if t.checkExpiration() {
+		err := fmt.Errorf("license is expired, valid license doesn't exist with user-id: %s, key: %s and platform uuid: %s", LCfg.Lcs.UserId, LCfg.Lcs.Key, LCfg.Lcs.PlatformUUID)
```
Comment (seswarrajan) [inline src/license/license.go:86]: Do we need to print key in the error ?
We need only user and uuid to generate new token. I think only those 2 are required.

**Review by seswarrajan — CHANGES_REQUESTED** [review]

File: src/license/license.go (hunk 1 above)
Comment (rajaSahil) [reply src/license/license.go:86]: Printed it so user can validate license key that they used while installing.

File: src/license/license.go [hunk 2]
```diff
@@ -59,13 +59,14 @@ func CheckLicenseSecret() error {
 		return errors.New("license secret doesn't exist for discovery-engine")
 	}
 
-	l := &License{
+	LCfg.Lcs = &License{
 		UserId: string(secret.Data["user-id"]),
```
Comment (vishnusomank) [inline src/license/license.go:69]: What if the secret exists and the required keys `user-id` or `key` is not found.? In this scenario, we can directly return an error and exit right?

Comment (rajaSahil) [reply src/license/license.go:69]: Ah, I didn't consider this edge case, thanks for pointing it out!

**Review by seswarrajan — APPROVED** [review]
LGTM

**Pull request merged**